### PR TITLE
1990 rjsf -- Benefits Selection

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2813

Old on the left, rjsf on the right.
image

I'm not fond of the spacing changes, so if we need to space out the checkboxes more, we can add classes to each checkbox for that.

### CODE REVIEW

**Comment by jbalboni** [comment]
@cvalarida Spacing looks ok, I think, except for between the blue box and the label.

File: src/js/edu-benefits/1990-rjsf/config/form.js [hunk 1]
```diff
@@ -61,10 +72,59 @@ const formConfig = {
...
+                'ui:title': benefitsLabels.chapter1606
+              },
+              chapter32: {
+                'ui:title': benefitsLabels.chapter32
+              }
+            }
           },
           schema: {
             type: 'object',
+            // TODO: Make one of these required; this isn't working like this
```
Comment (jbalboni) [inline src/js/edu-benefits/1990-rjsf/config/form.js]: What's not working about this?

**Review by jbalboni — APPROVED** [review]
Look fine to me, though I'm curious about the TODO comment.

File: src/js/edu-benefits/1990-rjsf/config/form.js (hunk 1 above)
Comment (cvalarida) [inline src/js/edu-benefits/1990-rjsf/config/form.js]: Oh, that's an old comment. It's working now.

**Pull request merged**